### ISSUE TITLE
Crash (Adapter 'NVIDIA GeForce GTX 1050' does not support the Reach profile)

### ISSUE THREAD

**Issue by MJLPRO** [Regalis11/Barotrauma#9515]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When opening barotrauma on steam I get an error message saying -
'Oops Barotrauma just crashed. - A crash report (crash report(4).log) (p.s. Its the 4th because this is the 4th time opening it) was saved in the root folder of the game' 
When closing this pop up it takes me to note pad -
Barotrauma seems to have crashed sorry for inconvenience
 A382F735B43D8E3FAD0FC906B3E782B4

Game version 0.18.12.0 (ReleaseWindows, branch release, revision 85532fbcbc)
Graphics mode: 0x0 (Windowed)
VSync OFF
Language: 
Selected content packages: None
Level seed: no level loaded
Loaded submarine: None
Selected screen: None
SteamManager initialized
- Basically saying it hasn't started up
Exception: Adapter 'NVIDIA GeForce GTX 1050' does not support the Reach profile. (Microsoft.Xna.Framework.Graphics.NoSuitableGraphicsDeviceException)
Target site: Void .ctor(Microsoft.Xna.Framework.Graphics.GraphicsAdapter, Microsoft.Xna.Framework.Graphics.GraphicsProfile, Microsoft.Xna.Framework.Graphics.PresentationParameters)

### Reproduction steps

1. Start Barotrauma 
2. Crash 

### Bug prevalence

Happens every time I play

### Version

0.18.12.0

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
Previously stated in description
```

**Comment by MJLPRO** [comment]
10th attempt - completely uninstalled from files then reinstalled no change.

**Comment by MJLPRO** [comment]
Fixed with a restart and a crash?
Welp

**Comment by Regalis11** [comment]
Thank you for the report!

Do you mean you were able to fix the issue now with a restart? A quick google search seems to suggest that the "Adapter does not support the Reach profile" error might be a driver issue that can be fixed with a reboot. You might also want to ensure your graphics drivers are up to date.
